csharp/README.md

```diff
@@ -17,15 +17,18 @@ To run the XenServer.NET examples, follow these steps:
 1. **Install Dependencies**:
    - Ensure that the XenServer.NET NuGet package is installed in your project. You can download the stable release from [xenserver.com/downloads](xenserver.com/downloads).
    - Alternatively, if you prefer using an unstable prerelease version, you can fetch it from the [Xen-API repository's release page](https://github.com/xapi-project/xen-api/releases).
+   - This will update the `<PackageReference>` entry in the `XenSdkSample.csproj` file.
 
 2. **Open Solution in Visual Studio**:
    - Open the `XenSdkSample.sln` solution file in Microsoft Visual Studio 2022.
 
 3. **Compile the Solution**:
    - Build the solution within Visual Studio to compile the example projects.
+   - You may need to change the `<RuntimeIdentifier>` depending on the OS you are using in the `XenSdkSample.csproj`. This can be `win-x64`, `linux-x64` etc.
```

Comment:
This is not needed, see [this comment](https://github.com/xenserver/xenserver-samples/pull/47/files#r2294388472).